contrib/actions/cds-python-pylint.hcl

```diff
@@ -0,0 +1,306 @@
+
+name = "CDS_PythonPylint"
+description = "Run pylint.
+By default, each pylint recommendation will be displayed as a Junit test.
+The short errors are fully contained inside the test title,
+an arrow (->) will be present if a part of the recommendation is displayed in
+the test body."
+
+// Requirements
+requirements = {
+    "Python-3.5" = {
+        type = "model"
```

Comment:
It's not a best practice to force a model in Public Actions.

You should add requirement binary virtualenv and python3.5, but not model.